lib/librte_hash/rte_cuckoo_hash.c

```diff
@@ -42,6 +42,13 @@ static struct rte_tailq_elem rte_hash_tailq = {
 };
 EAL_REGISTER_TAILQ(rte_hash_tailq)
 
+struct rte_hash_iterator_conflict_entries_state {
+	const struct rte_hash *h;
+	uint32_t              vnext;
+	uint32_t              primary_bidx;
+	uint32_t              secondary_bidx;
+};
+
```

Comment:
Add the following lines after the definition of `struct rte_hash_iterator_conflict_entries_state`:
```
#if sizeof(struct rte_hash_iterator_conflict_entries_state) > sizeof(struct rte_conflict_iterator_state)
#error struct rte_hash_iterator_conflict_entries_state is too large
#endif
```

Reply:
It seems that C doesn't support the preprocessor for ``sizeof()``. I also tried ``static_assert()``, however, it doesn't work. Do we need to do the comparison within ``rte_hash_iterator_conflict_entries_init()``?

Reply:
Let's ignore this issue. If DPDK developers care about it, they will tell us what they expect here.

By the way, I think `static_assert()` is C++, not C. In the kernel, we have `BUILD_BUG_ON()` to deal with these things, but it seems DPDK doesn't have a similar macro.

Reply:
Sounds good!